ocs_ci/templates/amq/examples/templates/cluster-operator/connect-s2i-template.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
```

Comment:
Lets clone the yaml files as we discussed before, we dont want to put so many yaml files in our repo.

Reply:
Done